File: examples/01_TTS/node/tts_watson.js
// Example TTS - "watson"

// required to run Child Process Applications
var exec = require('child_process').execSync;
// required to access utility functions such as "format"
var util = require('util');

function say(something){
  var APIkey = "YOUR_API_KEY";
  var url = "YOUR_API_URL";
  var data = encodeURIComponent(something);
  var voice = "voice=en-US_AllisonVoice";
  command = util.format('curl -X GET -u \"apikey:%s\" --output say.mp3 \"%s/v1/synthesize?accept=audio/mp3&text=\"%s\"&%s\" && afplay say.mp3 ', APIkey,url,data,voice);
  exec(command);
}

say(process.argv[2]); // Request the command line argument after the script name

File: examples/01_TTS/python/tts_watson.py
# Example TTS - "watson"

# required to access sytem commands and to run applications
import os
import sys
# required to format URL
import urllib

def say(something):
    APIkey = "YOUR_API_KEY"
    url = "YOUR_API_URL"
    data = urllib.quote_plus(something)
    voice = "voice=en-US_AllisonVoice"
    command = "curl -X GET -u \"apikey:{0}\" --output say.mp3 \"{1}/v1/synthesize?accept=audio/mp3&text=\"{2}\"&{3}\" && afplay say.mp3 ".format(APIkey,url,data,voice)
    os.system(command)

say(sys.argv[1]) # Request the command line argument after the script name
